### ISSUE TITLE
Is the eval loss of the original expert discriminator really ~0.25?

### ISSUE THREAD

**Issue by vokshin** [Rudrabha/Wav2Lip#318]
I could not train the Syncnet model for a long time to the required loss value ~0.25. Then I decided to check the loss value on the original weights of the val lrs2 sub-set and got a loss ~0.45. Should it be like this? Is it normal?

**Comment by Tony199110** [comment]
Yes, it is normal. It takes a really long time to train Syncnet, which is hard to converge.

**Comment by fuxuelinwudi** [comment]
> > Yes, it is normal. It takes a really long time to train Syncnet, which is hard to converge.
> 
> @Tony199110 How long does it take in your experience? I reproduce according to the README.md. But get ~0.71 loss at first step, ~0.69 at 140000 step with about 20 hours training.

i got same situation like yours. have you fix it yet?
